src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/models/rgw-storage-class.model.ts

```diff
@@ -84,6 +105,7 @@ export interface PlacementTarget {
   placement_id: string;
   tags?: string[];
   tier_type?: TIER_TYPE;
+  tier_config_rm: TierConfigRm;
```

Comment:
does actual payload really has this? or returns a reponse with this ? tier_config_rm , tier_config ., this is confusing as it seems like only tier_config should be there.